client/src/main/java/com/alibaba/nacos/client/config/impl/ClientWorker.java

```diff
@@ -775,48 +775,36 @@ public void notifyListenConfig() {
         
         @Override
         public void executeConfigListen() {
-            
+
```

Comment:
Please use nacos code style to reformat codes.

Reply:
> Please use nacos code style to reformat codes.

Okay, I corrected it